react-native/codepush-hockeyapp/appcenter-post-build.sh

```diff
@@ -0,0 +1,49 @@
+#!/usr/bin/env bash
+
+# Example: Install App Center CLI
+npm install -g appcenter-cli
+
+ENVIRONMENT=Development
+APP_NAME=OrganizationName/MyApp-iOS
+APP_PATH=myApp.ipa
+# HockeyApp-iOS-Dev
+HOCKEY_APP_ID=""
+HOCKEY_TOKEN=""
+APP_CENTER_TOKEN=""
+COMMIT_MESSAGE=$(git log -1 HEAD --pretty=format:%s)
+
+if [ "$APPCENTER_BRANCH" == "master" ]; then
+    ENVIRONMENT=Production
+    # HockeyApp-iOS-Prod
+    HOCKEY_APP_ID=""
+fi
+
+if [[ -z "$APPCENTER_XCODE_PROJECT" ]]; then
+    APP_NAME=OrganizationName/MyApp-Android
+    APP_PATH=app-debug.apk
+    # HockeyApp-Android-Dev
+    HOCKEY_APP_ID=""
+
+    if [ "$APPCENTER_BRANCH" == "master" ]; then
+        # HockeyApp-Android-Prod
+        HOCKEY_APP_ID=""
+    fi
+fi
+
+echo "**************** PUBLISH CHANGES WITH CODEPUSH ******************"
+echo "APP NAME               => $APP_NAME"
+echo "CURRENT ENVIRONMENT    => $ENVIRONMENT"
+echo "SELECTED RN PACKAGE    => $APPCENTER_REACTNATIVE_PACKAGE"
+echo "SELECTED XCODE PROJECT => $APPCENTER_XCODE_PROJECT"
```

Comment:
This would echo even when it's not set for Android projects - can we use a different variable here, i.e. `APPCENTER_ANDROID_MODULE` from [here](https://docs.microsoft.com/en-us/appcenter/build/custom/scripts/index#app-center-variables)?

Reply:
ok, so let me add the echo for Android